src/test/java/net/syscon/elite/api/resource/impl/OffenderMovementsResourceImplIntTest_scheduleCourtCaseHearing.java

```diff
@@ -25,17 +26,23 @@ public void schedules_court_hearing() {
         ));
 
         final var response = testRestTemplate.exchange(
-                "/api/bookings/-1/court-cases/-1/prison-to-court-hearings",
+                "/api/bookings/-2/court-cases/-2/prison-to-court-hearings",
                 HttpMethod.POST,
                 request,
                 new ParameterizedTypeReference<String>() {
                 });
 
-        assertThatJsonFileAndStatus(response, 201, "schedule_court_hearing.json");
+        assertThat(response.getStatusCode()).isEqualTo(CREATED);
+        assertThat(getBodyAsJsonContent(response)).extractingJsonPathNumberValue("$.id").isNotNull();
```

Comment:
This value is populated via a sequence on the DB so cannot guarantee the version.